CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+* New: Add backend support to allow for integration with WP's user profile ([#196](https://github.com/GlotPress/GlotPress-WP/issues/196))
+* New: Things: Introduce `after_delete()` method and new actions ([#294](https://github.com/GlotPress/GlotPress-WP/issues/294))
+* New: Translations: Include check button for case-sensitive search ([#312](https://github.com/GlotPress/GlotPress-WP/issues/312))
+* New: Java properties file support ([#297](https://github.com/GlotPress/GlotPress-WP/issues/297))
+* New: Standardize triggers in projects, translations and originals ([#294](https://github.com/GlotPress/GlotPress-WP/issues/294))
+* New: Use wp-dev-lib for development ([#320](https://github.com/GlotPress/GlotPress-WP/issues/320))
+* New: Public and private profiles do not have the same design ([#325](https://github.com/GlotPress/GlotPress-WP/issues/325))
+* New: Translation Sets: Validate slugs on saving ([#329](https://github.com/GlotPress/GlotPress-WP/issues/329))
+* Bugfix: Mark Sindhi locale as RTL ([#243](https://github.com/GlotPress/GlotPress-WP/issues/243))
+* Bugfix: Replace current_user_can( 'manage_options' ) with GP permissions ([#254](https://github.com/GlotPress/GlotPress-WP/issues/254))
+* Bugfix: Child projects are inaccessible if permalinks have trailing slash ([#265](https://github.com/GlotPress/GlotPress-WP/issues/265))
+* Bugfix: Various code cleanups to improve code quality ([#237](https://github.com/GlotPress/GlotPress-WP/issues/237))
+* Bugfix: Update readme files ([#228](https://github.com/GlotPress/GlotPress-WP/issues/228))
+* Bugfix: Tests: Remove unnecessary filters for `gp_set_cookie` ([#287](https://github.com/GlotPress/GlotPress-WP/issues/287))
+* Bugfix: Replace deprecated javascript back code ([#278](https://github.com/GlotPress/GlotPress-WP/issues/278))
+* Bugfix: [_]wp_specialchars() is deprecated/private and should be replaced ([#280](https://github.com/GlotPress/GlotPress-WP/issues/280))
+* Bugfix: Merge similar translation strings and avoid using HTML tags in translation strings ([#295](https://github.com/GlotPress/GlotPress-WP/pull/295))
+* Bugfix: Actions for translations have no `gp_` prefix ([#232](https://github.com/GlotPress/GlotPress-WP/issues/232))
+* Bugfix: `$original->validate()` fails if singular is '0' ([#301](https://github.com/GlotPress/GlotPress-WP/issues/301))
+* Bugfix: Auto generation of a project slug doesn't work correctly in some cases ([#328](https://github.com/GlotPress/GlotPress-WP/issues/328))
+* Bugfix: Suspend cache invalidation during original imports ([#332](https://github.com/GlotPress/GlotPress-WP/issues/332))
+* Bugfix: Do not allow translations with empty plurals ([#308](https://github.com/GlotPress/GlotPress-WP/pull/308))
+* Bugfix: Update schema definitions to work with WP's dbDelta() ([#343](https://github.com/GlotPress/GlotPress-WP/issues/343))
+* Bugfix: Translation Set: Use the correct select helper for the edit form ([#351](https://github.com/GlotPress/GlotPress-WP/pull/351))
```

Comment:
That was a bug in 6a8a67104825e78bcfe7328b50e2037d0de01f0f, doesn't need to be listed here.